how-to-extend/adapter-starterkit/gradle.properties

```diff
@@ -0,0 +1 @@
+version=1.5.1-SNAPSHOT
```

Comment:
This might be any other version to be honest :)
E.g. `1.0.0-SNAPSHOT`
We don't want to update it manually every release.